README.md

```diff
@@ -128,35 +128,35 @@ This is a detailed step by step instruction guide to setup your own e-commerce s
    ```sh
    npm install
    ```
-3. Open [firebase console](https://console.firebase.google.com/u/0/) with your Google account logged in and create a new project. Upgrade the project plan from spark to blaze by setting up your billing account. follow the instructions [here](https://docs.firerun.io/getting-started/upgrading-from-the-firebase-spark-plan-to-the-blaze-plan-tled).
-4. Hit on the firestore tab in the left drawer in the firebase console and create the firestore database with security rules in test mode. Follow this [guide](https://firebase.google.com/docs/firestore/quickstart#create).
+3. Rename a file named `.env.local.txt` to `.env.local` in root directory.
 
-5. Similarly, setup Authentication with Email/Password and Anonymous mode enabled. Also setup functions and storage with no changes made.
+4. Open [firebase console](https://console.firebase.google.com/u/0/) with your Google account logged in and create a new project. Upgrade the project plan from spark to blaze by setting up your billing account. follow the instructions [here](https://docs.firerun.io/getting-started/upgrading-from-the-firebase-spark-plan-to-the-blaze-plan-tled).
+5. Hit on the firestore tab in the left drawer in the firebase console and create the firestore database with security rules in test mode. Follow this [guide](https://firebase.google.com/docs/firestore/quickstart#create).
 
-6. Click on the web icon in project overview and create a web project with hosting enabled. and copy the Firebase SDK snippet in config mode from project settings. Replace these configs with the dummy ones below in `src/config/firebaseConfig.js`
+6. Similarly, setup Authentication with Email/Password and Anonymous mode enabled. Also setup functions and storage with no changes made.
+
+7. Click on the web icon in project overview and create a web project with hosting enabled. and copy the Firebase SDK snippet in config mode from project settings. Copy & paste each of these configs with the dummy ones present in `.env.local`.
    ```JS
     export var firebaseConfig = {
```

Comment:
I think this config variable is irrelevant now, 

can you replace this config variable with the below chunk in the readme
 
```
REACT_APP_FIREBASE_API_KEY=your_firebase_apikey
REACT_APP_FIREBASE_AUTH_DOMAIN=your_firebase_authDomain
REACT_APP_FIREBASE_DATABASE_URL=your_firebase_databaseUrl
REACT_APP_FIREBASE_PROJECT_ID=your_firebase_projectId
REACT_APP_FIREBASE_STORAGE_BUCKET=your_firebase_storageBucket
REACT_APP_FIREBASE_MESSAGING_SENDER_ID=your_firebase_messagingSenderId
REACT_APP_FIREBASE_APP_ID=your_firebase_appId
```

Reply:
Sure thing, I have updated the readme as requested. Please take a look :)